frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -50,6 +50,10 @@ import { BinaryUnit } from '../../form/size-unit-utils';
 import { StorageUISource } from './storage-ui-source';
 import { TemplateValidations } from '../../../utils/validations/template/template-validations';
 
+const CREATE = 'Create';
```

Comment:
can you please move these to `utils/strings` and reuse them?

Reply:
Done!